optimize-onnx/src/export.cpp

```diff
@@ -184,9 +184,16 @@ void encodeGraph(onnx::GraphProto * p_g, const std::shared_ptr<Graph> & g) {
       // to help symbolics do the right thing.
       continue;
     }
+    if (node->kind() == stringToSymbol("OnnxElidedInput")) {
```

Comment:
I would prefer to add a util function `is_elided` (or some other better name you can think of) in the Node class, instead of comparing node kind with this string value in all the places.